packages/ui/src/common/components/ContextMenu/ContextMenu.tsx

```diff
@@ -1,19 +1,20 @@
-import React, { useState } from 'react'
+import item from '@polkadot/ui-keyring/options/item'
+import React, { useEffect, useRef, useState } from 'react'
 import styled, { css } from 'styled-components'
 
 import { Animations, BorderRad, Colors, Shadows, Transitions } from '../../constants'
-import { ButtonGhost } from '../buttons'
+import { ButtonGhost, ButtonLink } from '../buttons'
 import { KebabMenuIcon } from '../icons'
 
 export interface ContextMenuProps {
-  children: React.ReactNode
+  items: { text: string; onClick: () => void }[]
```

Comment:
```suggestion
  items: ContextMenuItem[]
```

I think that `{}[]` is a bit harder to read, because the information about an array type is "hidden" after the object.